DTS-STN/canadian-dental-care-plan
# Splitting renew/ita/update-address to update-mailing-address and upda…

**Left some comments in the code to get your opinions**
### Description
Renewal Figma changed `update-address` into two pages: `update-mailing-address` and `update-home-address`.

Splitting addressInformation (state) should be considered:
![image](https://github.com/user-attachments/assets/228dc271-0e85-4c7a-aeb9-ed8f60b247a6)


### Related Azure Boards Work Items
AB#4919

### Screenshots (if applicable)


### Checklist


### Test Instructions
Navigate through the ITA flow and select 'update the mailing address' once on `confirm-address`

### Additional Notes
- Translations will be refactored depending on what we do with this PR.
